Libraries/Opc.Ua.Client/Session/Session.cs

```diff
@@ -3652,6 +3608,45 @@ public bool RemoveTransferredSubscription(Subscription subscription)
         }
         #endregion
 
+        #region Combined Browse/BrowseNext
+
+
+        /// <inheritdoc/>
+        public void ManagedBrowse(
+            RequestHeader requestHeader,
+            ViewDescription view,
+            IList<NodeId> nodesToBrowse,
+            uint maxResultsToReturn,
+            BrowseDirection browseDirection,
+            NodeId referenceTypeId,
+            bool includeSubtypes,
+            uint nodeClassMask,
+            out List<ReferenceDescriptionCollection> result,
+            out List<ServiceResult> errors,
+            bool excecuteDefensively = false)
+        {
+
+            var task = Task.Run(() =>
```

Comment:
is Task.Run needed?

Reply:
Not really sure. It works as expected so far, and I have issues to write it differently, so I would like to keep it that way, until someone explains to me how to improve on it.